### DISCUSSION TITLE
Deleting a GitHub account asks me to send feedback to a noreply email

### DISCUSSION THREAD

**Discussion by ssb22** [community/community#69812] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi, when I deleted a GitHub account, I got an email saying "We’re sorry to see you go. You can reply directly to this email if you have any questions or feedback. We’d love to hear from you." but the email came from noreply@github.com and I assume "noreply" means what it says....

(In my case, the reason why I deleted the account was because I found out that the Github Terms of Service does not allow more than one free account per person.  I had created a second account for a project which I wasn't yet sure I wanted to be associated with my real name, but when I found out it was against the Terms of Service, I immediately deleted that second account.)

**Comment by VedasreeM** [comment, marked as answer]
Hello,

Thank you for sharing your experience with us. It's understandable that you may have concerns regarding the email coming from [noreply@github.com](mailto:noreply@github.com).

In this case, 'noreply' typically means that the email address is not actively monitored for replies. However, you can still provide feedback or ask questions through GitHub's official channels.

Regarding your situation, it's great that you took the proactive step to align with GitHub's Terms of Service. If you have any further questions or need assistance with anything else, feel free to let me know.

**Comment by Uliana200407** [comment]
Hi
It's unfortunate to hear that you had to delete your GitHub account due to the Terms of Service limitations. Your feedback regarding the notification email and the "noreply" address is valid. "Noreply" typically implies that the email address is not monitored for incoming messages. However, GitHub often uses this address for automated notifications.

While you can't directly reply to the "noreply" email address, you can provide feedback and discuss your concerns through other channels. Here's what you can do:
Maybe you can go to github support - explain them the situation
